search/package.json

```diff
@@ -22,9 +21,11 @@
     "mathjs": "^7.2.0",
     "moment": "^2.25.3",
     "stream": "0.0.2",
+    "upng-js": "^2.1.0",
     "uuid": "^8.2.0"
   },
   "devDependencies": {
+    "aws-sdk": "^2.720.0",
```

Comment:
Will this really work only as a dev dependency?

Reply:
yes, the lambda environment provides it by default. See here:

https://stackoverflow.com/questions/60948911/can-aws-sdk-be-a-development-dependency-when-developing-lambdas-with-nodejs

and

https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html